lambdas/src/main/kotlin/scorcerer/utils/PointsCalculator.kt

```diff
@@ -0,0 +1,18 @@
+package scorcerer.utils
+
+import scorcerer.db.Prediction
+
+data class Result(val homeScore: Int, val awayScore: Int)
+
+object PointsCalculator {
+    fun calculatePoints(
+        prediction: Prediction,
+        result: Result,
+    ) = when {
+        prediction.awayScore == result.awayScore && prediction.homeScore == result.homeScore -> 5
+        prediction.homeScore < prediction.awayScore && result.homeScore < result.awayScore -> 2
+        prediction.homeScore > prediction.awayScore && result.homeScore > result.awayScore -> 2
+        prediction.homeScore == prediction.awayScore && result.homeScore == result.awayScore -> 2
```

Comment:
It might be nicer to write a function which computes the result (homeWin, awayWin, draw) from a prediction or result and then compares the results directly here